src/request/request-tab.js

```diff
@@ -45,12 +45,12 @@ module.exports.registerTab({
     component: WebServices
 });
 
-var Request = require('./components/request-detail-panel-request').Request;
```

Comment:
I don't see it in the branch; GitHub is showing the change as a rename/edit (as expected).